src/main/java/seedu/address/logic/parser/GroupCommandParser.java

```diff
@@ -38,6 +38,12 @@ public GroupCommand parse(String args) throws ParseException {
             throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, GroupCommand.MESSAGE_USAGE));
         }
 
+        if (arePrefixesPresent(argMultimap, PREFIX_NUSID)
+                && !arePrefixesPresent(argMultimap, PREFIX_GROUP)
+                && !arePrefixesPresent(argMultimap, PREFIX_TAG)) {
```

Comment:
Maybe we really need to create a common method in ParserUtils that accepts an array of args